templates/index.html

```diff
@@ -19,26 +16,25 @@
 			<pre> MULTILINGUAL <br><sub style="font-size:40px; color:whitesmoke; font-family:verde;">DICTIONARY </sub>      
                                                 <sup> English | Spanish | French | German | Japanese </sup></pre>
 		</h1>
-		
+		<form method="POST">
 		Language: 
 		<select name="language" id="language">
 		{% for language in language_list %}
 			<option value="{{language}}">{{language}}</option>
 		{% endfor %}
 		</select>
+	</form>
+		<br>
+		<form method="POST">
+			Part of speech: 
+			<select name="partOfSpeech" id="partOfSpeech">
+			{% for PartOfSpeech in part_of_speech_list %}
+				<option value="{{PartOfSpeech}}">{{PartOfSpeech}}</option>
+			{% endfor %}
+			</select>
+		</form>
 		<br><br>
-
-		<select name="partOfSpeech" id="partOfSpeech">
-			<option value="Noun">Noun</option>
-			<option value="Verb">Verb</option>
-			<option value="Adjective">Adjective</option>
-			<option value="Adverb">Adverb</option>
-		</select>
-		<br><br>
-		
-		<button onclick = showAlert()>
-			<a href="http://127.0.0.1:5000/log">Okay</a>
-		</button>
-				
 	</body>
-</html>	
+	<script>$(function() {$("#language").on("change", function(event){$.ajax({data: {language : $("#language").val()},type:"POST",url : "{{url_for('Query')}}", success: function(response) {$("#partOfSpeech").html(response);;;
```

Comment:
It would be more readable if we can indent this single line using tabs and new lines